plugins/com.python.pydev.analysis/plugin.xml

```diff
@@ -177,7 +177,7 @@
                menubarPath="org.python.pydev.ui.actions.menu/pydev"
                tooltip="Force a code analysis for the selected elements (recursive)">
          </action>
-         <visibility><objectState name="projectNature" value="org.python.pydev.pythonNature"/></visibility>
+         <visibility><objectState name="contentTypeId" value="org.python.pydev.pythonfile"/></visibility>
```

Comment:
Actually, those actions can be executed when a project/folder is selected... with this changes I don't think they'd work anymore right? The visibility should still enable them for any project/folder selected in a project with a pythonNature (for both cases).